src/modules/shortcut_guide/pch.h

```diff
@@ -1,4 +1,5 @@
 #pragma once
+#define NOMINMAX

```

Comment:
In the future, we could have `NOMINMAX` defined at .vcxproj level. Also, I think we should create a task to move all of the stuff we currently do into some new .vcxproj template, since at this point we have so many things we account for (compiler option, CI switches etc.) that it's getting unwieldy.

Reply:
@yuyoyuppe 
can you open an issue for that?